### DISCUSSION TITLE
How to save the results I want, not the abnormal ones

### DISCUSSION THREAD

**Discussion by zhenzi0322** [celery/celery#7679] [Q&A]
Exceptions in the task also when the results are saved, how to exclude this point

[a comment by an automated account is not shown]

**Comment by auvipy** [comment]
please ask your questions to the discussion forum not in issue tracker.

**Reply by zhenzi0322** [reply]
> please ask your questions to the discussion forum not in issue tracker.

How to save the final result instead of saving the exception message
